xarray/tests/test_groupby.py

```diff
@@ -1867,10 +1867,10 @@ def test_resample(
             "2000-01-01", freq="6h", periods=10, use_cftime=use_cftime
         )
 
-        def resample_as_pandas(array, *args, **kwargs):
+        def resample_as_pandas(array, *args, **kwargs) -> None:
```

Comment:
Remove typing from internal test function, as it actually does have a return return value.

```suggestion
        def resample_as_pandas(array, *args, **kwargs):
```

Reply:
Thanks for catching / fixing this @kmuehlbauer!

Reply:
Yeah, it was my suggestion to add this, wasn't it?